### PR TITLE
Update SVT-AV1 target quality params for v0.9.0

### LINKED ISSUE #556: --encoder svt-av1 --target-quality fails with v0.9.0
```
$ pkg info -x av1an svt-av1 ffmpeg vapoursynth vmaf
av1an-0.3.1
svt-av1-0.9.0
ffmpeg-4.4.1_4,1
vapoursynth-r54_1
vmaf-2.3.0_2

$ av1an --encoder svt-av1 --vmaf --target-quality 95 -i pedestrian_area_1080p25.y4m -o out.mp4
Scene detection
  00:00:01 [#######################################################################################] 100%  375/375 (255.48 fps, eta 0s) Queue 2 Workers 1 Passes 1
Params: --preset 4 --keyint 240 --rc 0 --crf 25 --tile-columns 1 --tile-rows 0
ERROR [av1an_core::target_quality] [chunk 1] encoder crashed: exit status: 1
stdout:

stderr:
        [SVT-Warning]: --keyint is now intra-period + 1!
        Svt[info]: -------------------------------------------
        Svt[info]: SVT [version]:       SVT-AV1 Encoder Lib v0.9.0
        Svt[info]: SVT [build]  :       FreeBSD Clang 11.0.1 (git@github.com:llvm/llvm-project.git llvmorg-11.0.1-0-g43ff75f2c3fe)      64 bit
        Svt[info]: LIB Build date: Jan 22 2022 13:45:23
        Svt[info]: -------------------------------------------
        Unprocessed tokens:  --tf-level  --palette-level  --umv  --ext-block  --hbd-md  --bipred-3x3  --pred-me  --compound  --sg-filter-mode  --enable-restoration-filtering  --cdef-level  --disable-dlf  --mrp-level  --enable-pic-based-rate-est  --enable-local-warp  --enable-global-motion  --enable-interintra-comp  --obmc-level  --rdoq-level  --filter-intra-level  --enable-intra-edge-filter

        Error in configuration, could not begin encoding! ...
        Run SvtAv1EncApp --help for a list of options
        SvtMalloc[info]: you have no memory leak

source pipe stderr:
        av_interleaved_write_frame(): Broken pipe
        Error writing trailer of pipe:: Broken pipe

ERROR [av1an_core::broker] [chunk 1] encoder crashed: exit status: 1
stdout:

stderr:
        [SVT-Warning]: --keyint is now intra-period + 1!
        Svt[info]: -------------------------------------------
        Svt[info]: SVT [version]:       SVT-AV1 Encoder Lib v0.9.0
        Svt[info]: SVT [build]  :       FreeBSD Clang 11.0.1 (git@github.com:llvm/llvm-project.git llvmorg-11.0.1-0-g43ff75f2c3fe)      64 bit
        Svt[info]: LIB Build date: Jan 22 2022 13:45:23
        Svt[info]: -------------------------------------------
        Unprocessed tokens:  --tf-level  --palette-level  --umv  --ext-block  --hbd-md  --bipred-3x3  --pred-me  --compound  --sg-filter-mode  --enable-restoration-filtering  --cdef-level  --disable-dlf  --mrp-level  --enable-pic-based-rate-est  --enable-local-warp  --enable-global-motion  --enable-interintra-comp  --obmc-level  --rdoq-level  --filter-intra-level  --enable-intra-edge-filter

        Error in configuration, could not begin encoding! ...
        Run SvtAv1EncApp --help for a list of options
        SvtMalloc[info]: you have no memory leak

source pipe stderr:
        av_interleaved_write_frame(): Broken pipe
        Error writing trailer of pipe:: Broken pipe

  00:00:00 [######################################################################################] 100%  375/375 (6897.58 fps, eta 0s) ```
```

### PR DESCRIPTION
**Pull request by redzic**
Should fix #556, although now target quality won't work for SVT-AV1 <0.9.0 as we use preset 12. Not sure if we should use preset 8 instead by default as it did before, as SVT-AV1 0.9.0 was released quite recently so people may not have had enough time to update to it yet

### CODE REVIEW

**Comment by shssoichiro** [comment]
Maybe we run a version check by parsing `SvtAv1Enc --help` to determine which to use?

**Comment by silverbacknet** [comment]
`--version` but yes, absolutely critical given the huge reorganization between 0.8.8 and 0.9.0. If you ever feel like switching to the API, it's very easy to get there too, via svt_av1_get_version().

**Comment by redzic** [comment]
@shssoichiro I've updated this PR to include a version check for SVT

File: av1an-core/src/encoder.rs [hunk 1]
```diff
@@ -25,6 +26,19 @@ pub enum Encoder {
   x265,
 }
 
+pub static USE_OLD_SVT_AV1: Lazy<bool> = Lazy::new(|| {
+  let version = Command::new("SvtAv1EncApp")
+    .arg("--version")
+    .output()
+    .unwrap();
+
+  if memchr::memmem::find(b"v0.9.0", &version.stdout).is_some() {
```
Comment (shssoichiro) [inline av1an-core/src/encoder.rs:35]: Clippy says "help: you can reduce it to: `!memchr::memmem::find(b"v0.9.0", &version.stdout).is_some()`"

**Comment by shssoichiro** [comment]
Do we also need to update the parameter validation?

**Comment by redzic** [comment]
The parameter validation is dynamic, i.e. it parses the correct parameters at runtime based on `--help` output, so it shouldn't need updating.

**Review by shssoichiro — APPROVED** [review]
Looks good assuming CI passes

**Pull request merged**